R/use_msrv.R

```diff
@@ -0,0 +1,54 @@
+#' Add MSRV to DESCRIPTION
+#' 
+#' @param version character scalar, the minimum supported Rust version
+#' @param path character scalar, path to folder containing DESCRIPTION file
+#' 
+#' @details
+#' It is assumed that MSRV is greater than or equal to `version`. The result is
+#' "SystemRequirements: Cargo (Rust's package manager), rustc >= `version`."
+#' 
```

Comment:
I think this section could have more details on how exactly CRAN plays with the MSRV and what the consequences for the users are. Here's a quick draft, it's just to give an idea:

> The Minimum Stable Rust Version is the lowest version of Rust that can compile your crate. It can be determined using [cargo-msrv](https://github.com/foresterre/cargo-msrv). Currently, CRAN uses the Rust version [insert version] and requires package developers to specify the MSRV of their crate in the DESCRIPTION.
>
> If the MSRV is lower or equal to the Rust version used by CRAN, then the Rust version won't be a problem (but you still need to check the other CRAN-related rules).
>
> If the MSRV is higher than the Rust version used by CRAN, then CRAN won't be able to compile your package and therefore provide binaries. This has implications for the end users of your package: they will have to install it from source, meaning that they will need to compile your package using a version of Rust that is higher or equal to your MSRV.

The wording may change, but I think there should be three pieces of information in this section:

* cargo-msrv
* the Rust version used by CRAN
* the implications if MSRV > Rust version on CRAN

Note that those paragraphs are based on my own understanding, some details might be wrong.